### ISSUE TITLE
TabView tab do not updates its label when opened from saved file

### ISSUE THREAD

**Issue by rat-moonshine** [prominic/Moonshine-IDE#390]
Recently during a demonstration by Andrii, I noticed that Organizer nodes were generating blank labelled entries and I choose to investigate. 

![image](https://user-images.githubusercontent.com/24370870/46124453-f4c4c880-c241-11e8-8de1-39c39e674baa.png)

I found that the file opened with un-labelled tabs to the `TabView` component. I'm not sure if an unlabelled tab is acceptable in a real-life scenario, but while playing with this I found following problem:

- Drag a TabView component to the editor
- Create multiple tabs to the component and remove all their labels
- Save the file and close it 
- Re-open the file, file opens with unlabelled tabs as expected
- Try to change the label to the tabs in its property editor
- TabView tabs do not updates with the label changes [Bug]

**Comment by rat-moonshine** [comment]
I have updated TabView with condition to generate some label when tab(s) has no label, to the Organizer.

**Comment by rat-moonshine** [comment]
For some reason when a `ButtonBarButton` generated with a Tab, and blank value associated to its label, `ButtonBarButton.displayText` width/height sets to 0/0. I noticed setting a valid text therefore to the ButtonBar never updates `ButtonBarButton.displayText` size properties (invalidation didn't seem works here).

I have applied a bit hacky way, but this works for now. The fix still need some fine tuning.
